### PR TITLE
Enable livereload to be port-forwarded

### PR DESCRIPTION
**Pull request by andreycizov**
Working on: https://github.com/dldl/sphinx-server/issues/13

### CODE REVIEW

**Comment by andreycizov** [comment]
Could I get a response on this and what needs to be done in order for this to be merged if the response is a no?

Thanks.

File: livereload/server.py [hunk 1]
```diff
@@ -224,17 +225,27 @@ def application(self, port, host, liveport=None, debug=None, live_css=True):
...
             live_reload_path = "/livereload.js?port={port}".format(port=liveport)
 
+        src_script = ' + window.location.hostname + "{path}">'.format(path=live_reload_path)
+
+        if override_endpoint_client:
+            src_script = (
+                ' + window.location.host + "/livereload.js?port="'
+                ' + window.location.port + "'
+                '>'
+            )
```
Comment (lepture) [inline livereload/server.py]: when `liveport` is not `None`, the port should be the `liveport`. You need to alter some logic above.

**Comment by lepture** [comment]
@andreycizov I've fixed it in a better way, can you check if it works?

**Comment by andreycizov** [comment]
@lepture perfect! Thank you.

**Pull request closed without merging**